src/Microsoft.Azure.ServiceBus/Management/QueueDescription.cs

```diff
@@ -283,6 +284,12 @@ public string UserMetadata
             }
         }
 
+        /// <summary>
+        /// List of properties that were retrieved using GetQueue but is not understood by this version of client is stored here.
+        /// These will be sent back to the service as-is when UpdateQueue is called on this QueueDescription.
```

Comment:
>  on this QueueDescription.

Did you mean the list will be sent back when an already retrieved `QueueDescription` will be used in `UpdateQueue` call?

Reply:
Fixed in #575